src/console.c

```diff
@@ -103,3 +103,18 @@ void console_init()
 	console_reset();
 	console_putstring("\nconsole: initialized\n");
 }
+
+void console_set_fgcolor(int r, int g, int b)
```

Comment:
These should probably be unsigned ints. Having negative values doesn't make a lot of sense.